src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,58 @@
+import { DraftClaimantResponse } from 'features/claimant-response/draft/draftClaimantResponse.ts'
+import { ResponseRejection } from 'claims/models/response/core/responseRejection.ts'
+import { FormaliseRepaymentPlanOption } from 'claimant-response/form/models/formaliseRepaymentPlanOption'
+import { CourtDetermination } from 'claims/models/response/core/courtDetermination.ts'
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+
+export class ClaimantResponseConverter {
+
+  public static covertToClaimantRespose (draftClaimantResponse: DraftClaimantResponse): any {
+
+    if (draftClaimantResponse.rejectionReason && draftClaimantResponse.rejectionReason.text.length > 0) {
```

Comment:
That will not work for settlement of admissions with immediate payments. Consider structuring logic around value of `SettleAdmitted`.

Reply:
Thanks, I am in the process of restructuring this.